util/llbutil/copyop.go

```diff
@@ -33,6 +33,11 @@ func CopyOp(srcState pllb.State, srcs []string, destState pllb.State, dest strin
 			// the filepath.Match syntax, so by simply creating a wildcard where the
 			// first letter needs to match the current first letter gets us the single
 			// match; and no error if it is missing.
+
+			//Normalize path by dropping './'
+			if src[0:2] == "./" {
+				src = src[2:]
+			}
```

Comment:
```suggestion
			// Normalize path by dropping "./"
			strings.TrimPrefix(src, "./")
```

(You'll need to add the matching `"strings"` import)